### PR TITLE
Implement Microservice

### PR DESCRIPTION
**Pull request by Elyas-Amini**

### CODE REVIEW

**Review by MahdiyarGHD — APPROVED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by MahdiyarGHD — APPROVED** [review]

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Clients/EasyMicroservices.SupportsMicroservice.Clients.csproj [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
+    <Platforms>AnyCPU;x64;x86</Platforms>
+    <Authors>EasyMicroservices</Authors>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Version>0.0.0.3</Version>
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Clients/EasyMicroservices.SupportsMicroservice.Clients.csproj]:
```suggestion
    <Version>0.0.0.1</Version>
```

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketDepartmentCreateRequestContract.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketDepartmentCreateRequestContract.cs:1]: Use the 'Create' prefix from the start of the class name
example :
CreateTicketDepartmentRequestContract

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketDepartmentUpdateRequestContract.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketDepartmentUpdateRequestContract.cs:1]: Use the 'Update' prefix from the start of the class name
example :
UpdateTicketDepartmentRequestContract

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketHistoryCreateRequestContract.cs.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketHistoryCreateRequestContract.cs.cs:1]: Use the 'Create' prefix from the start of the class name

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketHistoryUpdateRequestContract.cs.cs.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketHistoryUpdateRequestContract.cs.cs.cs:1]: Use the 'Update' prefix from the start of the class name

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketSupportTimeHistoryCreateRequestContract.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketSupportTimeHistoryCreateRequestContract.cs:1]: Use the 'Create' prefix from the start of the class name

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketSupportTimeHistoryUpdateRequestContract.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketSupportTimeHistoryUpdateRequestContract.cs:1]: Use the 'Update' prefix from the start of the class name

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketUpdateRequestContract.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/Contracts/Requests/TicketUpdateRequestContract.cs:1]: Use the 'Update' prefix from the start of the class name

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/DataTypes/StatusType.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/DataTypes/StatusType.cs:1]: Renames it to 'TicketStatusType'

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/DataTypes/SensitivityStatusType.cs
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Domain/DataTypes/SensitivityStatusType.cs:1]: renames it to TicketSensitivityStatusType

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by Hamed-Bagherpour — APPROVED** [review]

**Review by yousef-langaran — APPROVED** [review]

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Schemas/TicketSchema.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EasyMicroservices.SupportsMicroservice.Database.Schemas
+{
+    public class TicketSchema : IUniqueIdentitySchema, ISoftDeleteSchema, IDateTimeSchema
+    {
+        public string Title { get; set; }
+        public TicketStatusType Status { get; set; }
+        public TicketSensitivityStatusType sensitivityStatus { get; set; }
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Schemas/TicketSchema.cs]:
```suggestion
        public TicketSensitivityStatusType SensitivityStatus { get; set; }
```

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]

**Pull request merged**